### PR TITLE
Remove: CrystalDewWorld.CrystalDiskInfo version 8.8.9

### PR DESCRIPTION
**Pull request by coolplaylinbot**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/124663)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @CoolPlayLin,

For this PR, the error is because the host's SSL certificate expired. 
image

If the HTTPS is changed down to HTTP, the links still work.

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]
Close with reason: No need to remove

File: manifests/c/CrystalDewWorld/CrystalDiskInfo/8.8.9/CrystalDewWorld.CrystalDiskInfo.installer.yaml
Comment (CoolPlayLin) [inline manifests/c/CrystalDewWorld/CrystalDiskInfo/8.8.9/CrystalDewWorld.CrystalDiskInfo.installer.yaml:1]: Close with reason: No need to remove

File: manifests/c/CrystalDewWorld/CrystalDiskInfo/8.8.9/CrystalDewWorld.CrystalDiskInfo.locale.en-US.yaml
Comment (CoolPlayLin) [inline manifests/c/CrystalDewWorld/CrystalDiskInfo/8.8.9/CrystalDewWorld.CrystalDiskInfo.locale.en-US.yaml:1]: Close with reason: No need to remove

File: manifests/c/CrystalDewWorld/CrystalDiskInfo/8.8.9/CrystalDewWorld.CrystalDiskInfo.yaml
Comment (CoolPlayLin) [inline manifests/c/CrystalDewWorld/CrystalDiskInfo/8.8.9/CrystalDewWorld.CrystalDiskInfo.yaml:1]: Close with reason: No need to remove

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**